tests/wake-format/basic/stdout

```diff
@@ -558,61 +574,77 @@ def all variant =
     require Pass x =
         map (_ variant) targets
         | findFail
+
     Pass (flatten x)
 
 # Install wake into a target location
 def doInstall dest kind =
     require Pass variant = toVariant kind
     require Pass datfiles = sources "{@here}/share" `.*`
+
     def binfilesResult = all variant
     def releaseBin exe = installAs "{dest}/{replace `\.[^.]*$` '' exe.getPathName}" exe
+
     require Pass datinstall = findFailFn (installIn dest ".") datfiles
     require Pass readmeSource = source "README.md"
     require Pass readme = installIn "{dest}/share/doc/wake" "." readmeSource
     require Pass binfiles = binfilesResult
     require Pass bininstall = findFailFn releaseBin binfiles
+
     def outputs = readme, bininstall ++ datinstall
+
     Pass outputs
 
 # Replace @VERSION@ with 'release'
 def setVersion release file =
     require Pass in = source "{file}.in"
+
     def script =
         "%
         set -e
         sed "s/@VERSION@/%{release}/g" "%{in.getPathName}" > "%{file}.tmp"
         mv "%{file}.tmp" "%{file}"
         %"
+
     shellJob script (in, Nil)
     | getJobOutput
 
 # Create a release tarball
 def tarball Unit =
     def releaseResult = buildAs Unit
     def timeResult = buildOn Unit
+
     require Pass release = releaseResult
     require Pass time = timeResult
+
     # Create debian + RedHat package files
     require Pass changelog = setVersion release "debian/changelog"
     require Pass spec = setVersion release "wake.spec"
+
     # Identify those sources files to include in the tarball
     require Pass allSources = sources "." `.*`
+
     def srcs =
         allSources
         | filter (!matches `(debian/|wake.spec).*` _.getPathName) # omit packaging files
         | filter (!matches `(\.circleci/|\.github/).*|\.dockerignore` _.getPathName) # omit CI files
         | filter (!matches `(.*/)*\.gitignore` _.getPathName) # omit git files
+
+
     def Pair testPaths wakePaths = splitBy (matches `tests/.*` _.getPathName) srcs
```

Comment:
Is this double space deliberate?

Reply:
no, nice catch!

Reply:
sorry took a bit to  fix this because I had to  add a new api mnemonic. But ultimately its a net positive so glad to have it.

Should be good now